dockerfiles/cross-build/Dockerfile.debian-sid

```diff
@@ -0,0 +1,34 @@
+# pull in base + a minimal set of useful packages
+FROM debian:sid as debian-sid-build
+
+ARG GOLANG_VERSION=1.14.8
+ARG GOLANG_URLDIR=https://dl.google.com/go
+ARG CREATE_USER="test"
+ARG USER_OPTIONS=""
+ENV PATH /go/bin:/usr/local/go/bin:$PATH
+
+# pull in stuff for cgo
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        build-essential fakeroot devscripts \
+        bash git make sed debhelper wget ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN arch="$(dpkg --print-architecture)"; \
+    case "${arch##*-}" in \
+        amd64) goarch=linux-amd64;; \
+        i386) goarch=linux-386;; \
+        armhf) goarch=linux-armv6l;; \
+        ppc64el) goarch=linux-ppc64le;; \
+        s390x) goarch=linux-s390x;; \
+    esac; \
+    \
+    wget $GOLANG_URLDIR/go$GOLANG_VERSION.$goarch.tar.gz -O go.tgz && \
+    tar -C /usr/local -xvzf go.tgz && rm go.tgz && \
+    \
+    export PATH="/usr/local/go/bin:$PATH" && \
+    echo "PATH=/usr/local/go/bin:$PATH" > /etc/profile.d/go.sh && \
+    go version
+
+RUN [ -n "$CREATE_USER" -a "$CREATE_USER" != "root" ] && \
+    useradd -m -s /bin/bash $CREATE_USER $(echo $USER_OPTIONS | sed 's/__/ /g') || :
```

Comment:
The sed magic was to avoid even having to think about at all whether/how/if escaping of arguments with spaces is necessary to get them unaltered through all the hops. I'll check if we can get along without it.

Reply:
Mm, understood. But, I think this is unnecessary here. Thus, I believe it would be better to drop it because a similar practice is not used for other docker build-args - makes it more consistent and readable.

Reply:
#501 changes dockerfiles/the build script to not ignore errors and pass user ID option without sed trickery.